api/models/graphite.go

```diff
@@ -192,6 +192,8 @@ type GraphiteTagDelByQueryResp struct {
 type GraphiteTagTerms struct {
 	Tags []string `json:"tags"`
 	Expr []string `json:"expressions"`
+	From int64    `json:"from" form:"from"`
+	To   int64    `json:"to" form:"to"`
```

Comment:
do we need these form directives?

Reply:
I believe they are the same as the default.